pom.xml

```diff
@@ -5431,6 +5432,20 @@
         <version>${version.ch.obermuhlner}</version>
       </dependency>
 
+      <!-- used by DMN JS version for BigDecimal arithmetics -->
+      <dependency>
+        <groupId>org.treblereel.gwt</groupId>
+        <artifactId>gwt-bigmath</artifactId>
+        <version>${version.ch.obermuhlner.js}</version>
+      </dependency>
+
+      <!-- used by DMN for JS and Java compatible regular expressions -->
+      <dependency>
+        <groupId>org.gwtproject.regexp</groupId>
+        <artifactId>gwt-regexp</artifactId>
+        <version>HEAD-SNAPSHOT</version>
```

Comment:
The used version doesn't seem like sth ready for production. Also it should be maintained via property, instead of direct constant here.

Reply:
Yes, I asked the maintainer if he could do a release for us. If not, I need to fork it.